packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -72,6 +75,10 @@ export interface FileUploadValidationError {
     message: React.ReactNode;
 }
 
+interface RejectedFile {
+    file: FileWithFolderPath;
+}
```

Comment:
I plan to add an error message in the future but I'm not sure about the format yet, so I'll do that in a follow-up PR